proposals/ard_surface_reflectance.json

```diff
@@ -0,0 +1,77 @@
+{
+    "id": "ard_surface_reflectance",
+    "summary": "CARD4L compliant Surface Reflectance generation",
+    "description": "Computes CARD4L compliant surface reflectance values from optical input.",
+    "categories": [
+        "cubes",
+        "sar",
+        "ard"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "description": "The source data cube containing multi-spectral optical top of the atmosphere (TOA) reflectances. There must be a single dimension of type bands available.",
+            "name": "data",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "description": "The atmospheric correction method to use.",
+            "name": "athmosperic_correction_method",
+            "schema":  {
+                "type": [
+                    "string"
+                ],
+                "enum": [
+                    "FORCE",
+                    "iCor"
+                ]
+            }
+        },
+        {
+            "description": "The cloud detection method to use.",
+            "name": "cloud_detection_method",
+            "schema": {
+                "type": [
+                    "string"
+                ],
+                "enum": [
+                    "sen2cor",
+                    "fmask",
+                    "s2cloudless"
```

Comment:
force is missing?

Reply:
Not sure, I have not compiled this list of options. If you can replace the "?" with a "!" I'll add it ;-)

Reply:
@lforesta  I guess you can confirm this?

Reply:
I've just added it for now. If @lforesta can't confirm, we'll remove it again.
I also change the casing of the options to conform to the official casing (e.g. "Sen2Cor" instead of "sen2cor", "iCOR" instead of "iCor" etc)

Reply:
I've also added FORCE to cloud_detection, although it's likely not yet possible to use it standalone?!

Reply:
FORCE uses fmask for cloud detection (see https://force-eo.readthedocs.io/en/latest/howto/l2-ard.html#tut-ard , point 5). I have not seen a way to use this independently from the utility that generates ARD.
It must be possible to isolate the function that is used internally for this, but it would require quite some extra work I guess.

Reply:
I'll remove FORCE again, thank you.